src/core/geometry/qgsabstractgeometry.h

```diff
@@ -88,7 +88,13 @@ class CORE_EXPORT QgsAbstractGeometry
     //! Segmentation tolerance as maximum angle or maximum difference between approximation and circle
     enum SegmentationToleranceType
     {
+
+      /** Maximum angle between generating radii (lines from arc center
+       * to output vertices) */
       MaximumAngle = 0,
```

Comment:
That other format doesn't change a thing here. Do you see *any* enum ever documented in QGIS apidoc ?